src/Lex.x

```diff
@@ -59,6 +60,13 @@ tokens :-
   CALLER                                { mk CALLER }
   CALLVALUE                             { mk CALLVALUE }
   ORIGIN                                { mk ORIGIN }
+  BLOCKHASH                             { mk BLOCKHASH }
+  NUMBER                                { mk NUMBER }
```

Comment:
`BLOCKNUMBER` might be less confusing

Reply:
yeah, I suppose we can be a bit foreboding of that eip..